test/addMissingDependencies/uriParamsInterim.config.json

```diff
@@ -0,0 +1,69 @@
+{
+	"modules": {
+		"Replacement interim solution fix": {
+			"UriParameters with url param (fromQuery search)": {
+				"newExpressionArgsLength": 1,
+				"newExpressionArgs": ["window.location.href"],
+				"newExpressionCalleeName": "UriParameters",
+				"newVariableName": "UriParameters",
+				"functionName": "fromQuery",
+				"newArgs": ["window.location.search"],
+				"importPath": "sap/base/util/UriParameters",
+				"replacer": "Call",
+				"finder": "NewExpressionFinder",
+				"extender": "LeaveImport",
+				"version": "^1.58.0"
+			},
+			"UriParameters with url param (fromQuery href)": {
+				"newExpressionArgsLength": 1,
+				"newExpressionArgs": ["window.location.search"],
+				"newExpressionCalleeName": "UriParameters",
+				"newVariableName": "UriParameters",
+				"functionName": "fromQuery",
+				"newArgs": ["window.location.search"],
+				"importPath": "sap/base/util/UriParameters",
+				"replacer": "Call",
+				"finder": "NewExpressionFinder",
+				"extender": "LeaveImport",
+				"version": "^1.58.0"
+			},
+			"UriParameters with url param (fromQuery empty)": {
+				"newExpressionArgsLength": 0,
+				"newExpressionCalleeName": "UriParameters",
+				"newVariableName": "UriParameters",
+				"functionName": "fromQuery",
+				"newArgs": ["window.location.search"],
+				"importPath": "sap/base/util/UriParameters",
+				"replacer": "Call",
+				"finder": "NewExpressionFinder",
+				"extender": "LeaveImport",
+				"version": "^1.58.0"
+			},
+			"UriParameters with query param (fromURL)": {
+				"newExpressionArgsLength": 1,
+				"newExpressionCalleeName": "UriParameters",
+				"newVariableName": "UriParameters",
+				"functionName": "fromURL",
+				"importPath": "sap/base/util/UriParameters",
+				"replacer": "Call",
+				"finder": "UriParametersWithURLFinder",
+				"extender": "LeaveImport",
+				"version": "^1.58.0"
+			}
+		}
```

Comment:
looks like a copy of the "real" config. Could it be reused or did I miss a significant difference between the two?

Reply:
Indeed it is a copy of a fragment in the real config, to isolate the test data. Using the "big" real config would be too much imho and too confusing.
Having this separated makes it easier to read because it is less.

Reply:
As long as you can keep it consistent :-)